src/link/Coff.zig

```diff
@@ -1419,39 +1410,37 @@ pub fn updateDeclExports(
 
     const ip = &mod.intern_pool;
 
-    if (build_options.have_llvm) {
```

Comment:
I think this was a bug before and this should have `if (options.use_llvm)` gating so that we don't run this in self-hosted native backends route but let's see what the CI says about this.